modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayTemplate.testcase

```diff
@@ -97,25 +97,153 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-163909 - Verify if that can Activate Disabled Template"
+	@priority = "5"
+	test CanActivateDisabledTemplate {
+		task ("Given that the user adds a Template with status active") {
+			Testray.goToProjectPage(projectName = "Projects_1 Project");
+
+			TestrayContextMenu.clickContextMenuProjects(name = "Routines_1 Routine");
+
+			TestrayContextMenu.selectItemFromProjectsContextMenu(item = "Select Default Environment Factors");
+
+			TestrayBuild.selectDefaultEnvironmentFactors(
+				label = "Application Server",
+				label1 = "Browser",
+				listCategory = "Application Server,Browser",
+				listOption = "JBoss 6.1",
+				listOption1 = "Chrome x");
+
+			TestrayRoutines.goToRoutine(routineName = "Routines_1 Routine");
+
+			TestrayTemplate.addTemplate(
+				description = "Template Description",
+				dropdownItem = "New Template",
+				fieldName = "Build_Template",
+				gitHash = "12345",
+				routine = "Routines_1 Routine");
+
+			Testray.kebabMenuOption(kebabOption = "Manage Templates");
+
+			TestrayTemplate.viewTemplateDetails(
+				nameTemplate = "Build_Template",
+				statusTemplate = "Active");
+		}
+
+		task ("When changed the status of deactivate for Active") {
+			TestrayContextMenu.clickContextMenuProjects(name = "Build_Template");
+
+			TestrayContextMenu.selectItemFromProjectsContextMenu(item = "Deactivate");
+
+			Refresh();
+
+			TestrayTemplate.viewTemplateDetails(
+				nameTemplate = "Build_Template",
+				statusTemplate = "Deactive");
+
+			TestrayContextMenu.clickContextMenuProjects(name = "Deactive");
+
+			TestrayContextMenu.selectItemFromProjectsContextMenu(item = "Active");
+
+			Refresh();
```

Comment:
@joelrocha68 why do we need this refresh?

Reply:
Because the status Active only appears when refreshing the page.